graphdatascience/query_runner/arrow_query_runner.py

```diff
@@ -39,23 +39,27 @@ def run_query(self, query: str, params: Dict[str, Any] = {}) -> DataFrame:
         if "gds.graph.streamNodeProperty" in query:
             graph_name = params["graph_name"]
             property_name = params["properties"]
-            return self._run_arrow_property_get(graph_name, "NODE", property_name)
+            return self._run_arrow_property_get(
+                graph_name, "gds.graph.streamNodeProperty", {"nodeProperty": property_name}
+            )
         elif "gds.graph.streamRelationshipProperty" in query:
             graph_name = params["graph_name"]
             property_name = params["properties"]
-            return self._run_arrow_property_get(graph_name, "RELATIONSHIP", property_name)
+            return self._run_arrow_property_get(
+                graph_name, "gds.graph.streamRelationshipProperty", {"relationshipProperty": property_name}
```

Comment:
we changed this to `relationship_property` and `relationship_types`

Reply:
Looks like it's `relationshipProperty` and `relationshipLabels`. Updated it to that and works now.